sound/soc/sof/sof-client-ipc-test.c

```diff
@@ -33,8 +33,12 @@ struct sof_ipc_client_data {
 
 static int sof_ipc_dfsentry_open(struct inode *inode, struct file *file)
 {
+	struct sof_client_dev *cdev = inode->i_private;
 	int ret;
 
+	if (sof_client_get_fw_state(cdev) == SOF_FW_CRASHED)
+		return -ENODEV;
```

Comment:
what happened before this test was introduced? Did we get an error or a kernel oops?